### PR TITLE
Issue#771

### PR DESCRIPTION
**Pull request by j0rt3g4**
Added 2 important links into the "More Information Section" with 2 links added to solve this issue #771

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: SecurityCompliance/retention-policies.md [hunk 1]
```diff
@@ -370,5 +372,7 @@ To use the retention policy cmdlets, you need to:
 ## More information
 
 - [Overview of labels](labels.md)
+- [SharePoint Online Limitations](https://docs.microsoft.com/office365/servicedescriptions/sharepoint-online-service-description/sharepoint-online-limits)
```
Comment (JohanFreelancer9) [inline SecurityCompliance/retention-policies.md]:
```suggestion
- [SharePoint Online Limits](https://docs.microsoft.com/office365/servicedescriptions/sharepoint-online-service-description/sharepoint-online-limits)
```

File: SecurityCompliance/retention-policies.md [hunk 2]
```diff
@@ -370,5 +372,7 @@ To use the retention policy cmdlets, you need to:
...
+- [Microsoft Teams limits](https://docs.microsoft.com/en-us/microsoftteams/limits-specifications-teams)
```
Comment (JohanFreelancer9) [inline SecurityCompliance/retention-policies.md]:
```suggestion
- [Limits and specifications for Microsoft Teams](https://docs.microsoft.com/en-us/microsoftteams/limits-specifications-teams) 
```

**Comment by scanum** [comment]
@stephow-MSFT PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by j0rt3g4** [comment]
I've added the suggestions from @JohanFreelancer9 . Thank you! so It's ready for the final checks.

**Pull request merged**